### DISCUSSION TITLE
GitHub Actions Importer for TeamCity

### DISCUSSION THREAD

**Discussion by jaswalkiranavtar** [github/gh-actions-importer#158] [Feedback]
Hi,

My organization is currently using Bitbucket for version control and Jetbrains TeamCity for CICD.  We are exploring GitHub and GitHub Actions to be used as next gen systems.

I am wondering, are you working on or planning to add support for importing TeamCity pipelines to GitHub Actions Importer in near future?  

Also is GitHub Actions Importer an open source tool? If yes, what is the URL for code repo? We would like to familiarize with source code for existing importers like jenkins and may even contribute to TeamCity importer.

Regards,
Gaurav

**Comment by rkrishnasanka** [comment]
Hey @jaswalkiranavtar  !

I don't know if you guys figured out anything for this. But we are actually building some tooling to import TeamCity pipelines into Github actions. 

If you guys figured out how to do it, I'd love to know what you all did. We have a system that we are building that helps with these migrations and ensuring compliance.
